planning:
  euclidean_distance_cutoff: 3.0 # [m]
  safe_obstacle_distance: 1.0 # [m]
  bumper_distance_factor: 0.9 # virtual bumper will intervene at (safe_obstacle_distance * bumper_distance_factor)
  unknown_is_occupied: false
  navigation_tolerance: 0.4 # [m]
  min_altitude: 1.0 # [m]
  max_altitude: 10.0 # [m]
  max_goal_distance: 50.0 # [m]
  distance_penalty: 1.0
  greedy_penalty: 1.0
  planning_tree_resolution: 0.4
  max_waypoint_distance: 0.4 # [m] or set negative to allow maximal segment length
  max_yaw_step: 0.1 # [rad]
  planning_timeout: 4.5 # [s]
  replanning_limit: 3 # number of repeated attempts
  waypoint_replanning_limit: 5 # If it tries to go same waypoint more than this times, it will give up
  replanning_distance: 15.0 # [m] Euclidean distance from first waypoint
  override_previous_commands: true # new GOTO will override all previous actions
  main_update_rate: 10.0 # [Hz] 

visualization:
  visualize_planner: true
  show_unoccupied: false
  tree_points_scale: 0.22
  field_points_scale: 0.22
  expansions_points_scale: 0.15
  path_points_scale: 0.18
  goal_points_scale : 0.28

bumper:
  enabled: false
